src/class_resolver/api.py

```diff
@@ -33,11 +33,13 @@ def __init__(
         :param classes: A list of classes
         :param base: The base class
         :param default: The default class
-        :param suffix: The optional shared suffix of all classes
+        :param suffix: The optional shared suffix of all classes. If None, use the base class' name for it.
         :param synonyms: The optional synonym dictionary
         """
         self.base = base
         self.default = default
+        if suffix is None:
+            suffix = normalize_string(base.__name__, suffix=None)
```

Comment:
I think `suffix=None` is a bit redundant. Why include it?

Reply:
ae25051